master/internal/experiment.go

```diff
@@ -484,25 +569,26 @@ func (e *experiment) isBestValidation(msg searcher.CompletedMessage) bool {
 	return isBest
 }
 
-func (e *experiment) updateState(ctx *actor.Context, state model.State) {
-	if wasPatched, err := e.Transition(state); err != nil {
+func (e *experiment) updateState(ctx *actor.Context, state model.State) (wasPatched bool, err error) {
```

Comment:
question: Why are we returning wasPatched? It seems unused.

Reply:
was using it earlier. since we don't care to report it I'll remove it